x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -171,7 +171,7 @@ export const ML_GROUP_IDS = [ML_GROUP_ID, LEGACY_ML_GROUP_ID];
 /*
   Rule notifications options
 */
-export const ENABLE_CASE_CONNECTOR = true;
+export const ENABLE_CASE_CONNECTOR = false;
```

Comment:
@XavierM is there a better way to do this? could we tile this value to the `subCasesEnabled` feature flag?

Reply:
I think so we do not need to have two for the same thing, `SUB_CASES_ENABLED` or `CASE_CONNECTOR_ENABLED` should be sufficient.

Reply:
Hmm I think we might need two flags because the sub cases feature flag is stored in the `config.ts` and is passed into the case api integration tests. I don't think we can use that value in the security solution right? The flag needs to exist in both cases and security solution right?

Reply:
I moved this flag to the cases plugin so we only have one flag now 👍